lms/envs/aws.py

```diff
@@ -176,6 +176,7 @@
 EDXMKTG_USER_INFO_COOKIE_NAME = ENV_TOKENS.get('EDXMKTG_USER_INFO_COOKIE_NAME', EDXMKTG_USER_INFO_COOKIE_NAME)
 
 LMS_ROOT_URL = ENV_TOKENS.get('LMS_ROOT_URL')
+ENTERPRISE_API_URL = ENV_TOKENS.get('ENTERPRISE_API_URL', LMS_ROOT_URL + '/enterprise/api/v1/')
```

Comment:
@bradenmacdonald 
It looks like LMS_ROOT_URL was not in my lms json config file, /edx/app/edxapp/lms.env.json. I have not modified this file.  I am going to attempt to rebuild my devstack after my current Story is complete.